src/include/mpir_handlemem.h

```diff
@@ -418,6 +424,8 @@ static inline void MPIR_Handle_obj_free_unsafe(MPIR_Object_alloc_t * objmem, voi
             MPIR_Handle_free(objmem);
         }
     }
+    /* Currently enable_gc must be enabled for MPI_Info. */
```

Comment:
`enable_gc` has been renamed

Reply:
Thanks! Yes it has been renamed. It is renamed to `is_info`.
`enable_gc` does not appear in this PR.